Makefile

```diff
@@ -394,8 +394,9 @@ easyjson-gen:
 
 containerProtoVer=v0.2
 containerProtoImage=tendermintdev/sdk-proto-gen:$(containerProtoVer)
-containerProtoGen=cosmos-sdk-proto-gen-$(containerProtoVer)
-containerProtoGenSwagger=cosmos-sdk-proto-gen-swagger-$(containerProtoVer)
+containerSuffix=$(shell echo $(CURDIR) | sed 's/\//./g')
```

Comment:
The path may contain bad characters (e.g. space), so it would be better to replace them. e.g. `sed 's/[^[:alnum:]_-]/./g'`